src/applications/vaos/tests/e2e/va-request.cypress.spec.js

```diff
@@ -127,9 +127,9 @@ describe('VAOS request flow', () => {
       cy.findByLabelText(/Sidney/).click();
     });
   });
-  it('should submit form successfully for a user with single system and enabled facility', () => {
+  it('should submit form successfully for a user with multi system including a Cerner facility', () => {
     initAppointmentListMock();
-    initVARequestMock({ facilityPageV2Enabled: false });
+    initVARequestMock({ cernerUser: true });
```

Comment:
fix the test to accept a variable which assigns it as cerner location similar to `va-appointment.cypress.spec.js`